jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -347,16 +347,13 @@ public void testJibTaskGroupIsSet() {
 
   @Test
   public void testLazyEvalForImageAndTags() {
-    // TODO: Pass in `-Djib.console=plain` as argument for build and remove filtering for cyan
-    // coloring regex once [#2764](https://github.com/GoogleContainerTools/jib/issues/2764) is
-    // submitted.
     UnexpectedBuildFailure exception =
         assertThrows(
-            UnexpectedBuildFailure.class, () -> testProject.build(JibPlugin.BUILD_IMAGE_TASK_NAME));
+            UnexpectedBuildFailure.class,
+            () -> testProject.build(JibPlugin.BUILD_IMAGE_TASK_NAME, "-Djib.console=plain"));
 
     String output = exception.getBuildResult().getOutput().trim();
```

Comment:
nit: while we're at it, since we use `assertThat().contains()`,
```suggestion
    String output = exception.getBuildResult().getOutput();
```